roles/copy-keys/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Additional public keys to be authorized to access the cluster nodes
+  authorized_key:
+    key: "{{ lookup('file', 'public_keys/' + item) }}"
+    user: "{{ ansible_user }}"
+    state: present
+  loop:
+  - stig
```

Comment:
Could you make this so that it just adds all keys in the folder so it doesn't need updating everytime we need to add/remove a key?

Reply:
Will this break if there's no public_keys folder

Reply:
shouldn't do, there is an ignore_errors flag